titus-server-gateway/src/main/java/com/netflix/titus/gateway/service/v3/internal/TaskDataInjector.java

```diff
@@ -182,11 +205,11 @@ private long getTaskRelocationTimeout() {
 
     private Task newTaskWithContainerState(Task task) {
         return task.toBuilder().setStatus(TaskStatus.newBuilder()
-                    .addAllContainerState(kubeApiConnector.getContainerState(task.getId()))).build();
+                .addAllContainerState(kubeApiConnector.getContainerState(task.getId()))).build();
```

Comment:
I think this line nukes out whatever original status stuff we had when we do `setStatus` to this new status we build! (which only has ContainerState, and none of whatever we originally had)

Reply:
Should be `task.toBuilder().setStatus(task.getStatus().toBuilder()` I think

Reply:
https://github.com/Netflix/titus-control-plane/pull/1189